src/App.js

```diff
@@ -1,13 +1,35 @@
-import logo from "./logo.svg";
-import "./App.css";
-
+import React, { useState } from "react";
 function App() {
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>Learn react in the gym</p>
-      </header>
+    <div className="flex justify-center">
+      <div className="flex flex-col gap-3 w-1/5 mt-5">
+        <input
+          type="text"
+          className="input--style"
+          placeholder="First name"
+          value={firstName}
+          onChange={(v) => setFirstName(v.target.value)}
```

Comment:
your params should describe what it is doing, that means you can give you parameter a meaningful name